### ISSUE TITLE
encoded string missing <CR><LF>

### ISSUE THREAD

**Issue by phryniszak** [101100/nmea-simple#26]
Firstly, thank you for providing this library. I am not using it as whole but rather incorporating specific files. This may be one reason why encoding doesn't work for me. However, it seems that every encoded string is missing the CR LF at the end. Once I added it to the string, the module started responding to it.
Also encoding tests are ignoring that NMEA0183 full sentence should end with CR LF.

**Comment by 101100** [comment]
Good point. I confirmed with the [NMEA documentation by ESR](https://gpsd.gitlab.io/gpsd/NMEA.html#_nmea_encoding_conventions) and that is mentioned there as well. I see that the parsing example uses CR LF to split the input and there isn't a sending packet example so it isn't clear that you'd need to add that on a packet. Also the parsing code should probably be expanded to allow accepting packets with the CR LF terminator too.

**Comment by phryniszak** [comment]
And one more thing. It would be good to mention in readme which NMEA version this library is supporting (looks like v3.01). In my case module interpetter is V4.10 which in case GSV command gives one additional field compared to v3.01 that breaks correct satelite indexing - returns one additional with undefined members.
